core/node/fee_model/src/l1_gas_price/gas_adjuster/tests.rs

```diff
@@ -32,97 +32,92 @@ fn samples_queue() {
 #[test_casing(2, [L1BatchCommitmentMode::Rollup, L1BatchCommitmentMode::Validium])]
 #[tokio::test]
 async fn kept_updated(commitment_mode: L1BatchCommitmentMode) {
-    let eth_client = MockEthereum::builder()
-        .with_fee_history(vec![0, 4, 6, 8, 7, 5, 5, 8, 10, 9])
-        .with_excess_blob_gas_history(vec![
-            393216,
-            393216 * 2,
-            393216,
-            393216 * 2,
-            393216,
-            393216 * 2,
-            393216 * 3,
-            393216 * 4,
-        ])
-        .build();
+    // Helper macro to read a value from adjuster
+    macro_rules! read {
```

Comment:
Nit: Similar to the other line comment, I *think* this can be expressed as a generic function as well.